### DISCUSSION TITLE
Failed to load API definition. no Curso  Todo App: API com ASP.NET Core, CQRS e EF Core

### DISCUSSION THREAD

**Discussion by amarinafroes** [balta-io/forum#319] [Tira Dúvidas] [answered]
Bom dia @andrebaltieri.
Na aula de  Testando a API, ao usar o comando dotnet watch run, no terminal, o mesmo abriu uma pagina no meu navegador,
![image](https://user-images.githubusercontent.com/89412365/235666362-b5e19fb6-608d-4689-9132-a885315f8896.png)
única coisa que fiz de diferente no curso é acrescentar as linhas de código que mencionou no curso no Program.cs ao invés do Stutup.cs, pois no meu caso não apareceu a mesmo para mim acredito que por conta de uma atualização, porem segue meu código Program.cs:

using Microsoft.EntityFrameworkCore;
using Task.Domain.Handlers;
using Task.Domain.Infra.Contexts;
using Task.Domain.Infra.Repositories;
using Task.Domain.Repositories;

var builder = WebApplication.CreateBuilder(args);

// Add services to the container.

builder.Services.AddControllers();
// Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
builder.Services.AddEndpointsApiExplorer();
builder.Services.AddSwaggerGen();

builder.Services.AddDbContext<DataContext>(opt => opt.UseInMemoryDatabase("Database"));
//builder.Services.AddDbContext<DataContext>(opt => opt.UseSqlServer(Configuration.GetConnectionString("connectionString")));

builder.Services.AddTransient<ITaskRepository, TaskRepository>();
builder.Services.AddTransient<TaskHandler, TaskHandler>();

var app = builder.Build();

// Configure the HTTP request pipeline.
if (app.Environment.IsDevelopment())
{
    app.UseSwagger();
    app.UseSwaggerUI();
}

app.UseHttpsRedirection();

app.UseRouting();

app.UseCors(x => x
    .AllowAnyOrigin()
    .AllowAnyMethod()
    .AllowAnyHeader());

app.UseAuthentication();

app.UseAuthorization();

app.UseEndpoints(endpoints => 
{
    endpoints.MapControllers();
});

app.Run();

**Comment by amarinafroes** [comment, marked as answer]
Descobri o erro, por favor desconsiderar

**Reply by andrebaltieri** [reply]
🚀
